src/topnav.component.tsx

```diff
@@ -75,18 +80,25 @@ export class Topnav extends React.Component<Props> {
             >
               <ul className="navbar-nav">
                 {this.props.links.map((link, index) => {
-                  
-                  const linkClass = cx('nav-item', { [link.class]: !!link.class })
-                  
-                  var iconStr = {}
-                  if(link.icon) iconStr = <FontAwesomeIcon icon={link.icon} size="xs" />
+                  const linkClass = cx('nav-item', {
+                    [link.class]: !!link.class
+                  })
+
+                  var iconStr = null
```

Comment:
This is the only change (rest is code formatting), i.e. I changed `var iconStr = {}` to `var iconStr = null`